venia-e2e-tests/cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "baseUrl": "https://develop.pwa-venia.com/",
```

Comment:
Is there a way we can make this react to the current instances URL? For instance when running the tests in CI against a deployed PR we'll need to modify this URL.

Reply:
It has to be configurable. Definitely doable.